src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java

```diff
@@ -307,79 +309,95 @@ public void update() {
 				Matcher activeEffectsMatcher = PATTERN_ACTIVE_EFFECTS.matcher(line);
 				if (activeEffectsMatcher.matches()) {
 					foundGodPotText = true;
-					String[] godpotRemaingTimeUnformatted = activeEffectsMatcher.group(1).split(":");
 					long godPotDuration = 0;
 					try {
-						int i = 0;
-						if (godpotRemaingTimeUnformatted.length == 4) {
-							godPotDuration =
-								godPotDuration + (long) Integer.parseInt(godpotRemaingTimeUnformatted[i]) * 24 * 60 * 60 * 1000;
-							i++;
-						}
-						if (godpotRemaingTimeUnformatted.length >= 3) {
-							godPotDuration =
-								godPotDuration + (long) Integer.parseInt(godpotRemaingTimeUnformatted[i]) * 60 * 60 * 1000;
-							i++;
-						}
-						if (godpotRemaingTimeUnformatted.length >= 2) {
-							godPotDuration = godPotDuration + (long) Integer.parseInt(godpotRemaingTimeUnformatted[i]) * 60 * 1000;
-							i++;
-						}
-						if (godpotRemaingTimeUnformatted.length >= 1) {
-							godPotDuration = godPotDuration + (long) Integer.parseInt(godpotRemaingTimeUnformatted[i]) * 1000;
+						long godpotRemainingTime;
+							for (int i = 1; i < activeEffectsMatcher.groupCount(); i += 2) {
+								godpotRemainingTime = Integer.parseInt(activeEffectsMatcher.group(i));
```

Comment:
NPE

Reply:
I have no idea why, but suddenly the regex returns two more groups that contain only nulls
![image](https://user-images.githubusercontent.com/25665974/201472525-9bec7a9f-6f17-4b0c-b9f3-f4cce979f664.png)
![image](https://user-images.githubusercontent.com/25665974/201472538-d90a53bb-122f-4c4b-9327-2cca4162bd66.png)

Reply:
Looks like Hypixel changed the format again?

Reply:
https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/441